trinity/utils/filesystem.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+
+def is_same_path(p1, p2):
+    n_p1 = os.path.abspath(os.path.expanduser(p1))
+    n_p2 = os.path.abspath(os.path.expanduser(p2))
```

Comment:
*I'm rambling to myself, below. I think I resolved this to: take no action, but just in case I missed something...*

> `expandvars` doesn't make sense here, right? Anything supplied with a command line argument or variable with something like `$HOME` would get expanded before we see it. Plus, `expandvars` runs the risk of someone actually having a folder called `$HOME` and messing things up. Ok, I talked myself out of it making sense.